lib/Preloader/Preloader.js

```diff
@@ -1,16 +1,29 @@
 import React, { memo } from 'react';
+import PropTypes from 'prop-types';
 
 import {
   Loading,
   Layout,
 } from '@folio/stripes/components';
 
-export const Preloader = memo(({ message }) => (
+export const Preloader = memo(({
+  message,
+  size,
+  preloaderClassName,
```

Comment:
Please add tests to cover these newly added props with tests.

For each of them, it can be two tests to cover the default behavior (without props provided explicitly) and with props provided.

Reply:
Done